windows/installer-source/KaliteSetupScript.iss

```diff
@@ -242,14 +242,42 @@ begin
     end;
 end;
 
-procedure HandleUpgrade(targetPath : String);
+procedure SelectContentFolder;
 var
-    prevVerStr : String;
     userKaliteContent: String;
     systemKaliteDir: String;
     userPath : String;
     contentPath : String;
-    contentDb : String;
+    retCode: Integer;
+begin
+    userPath := ExpandConstant('{%USERPROFILE}')
+    contentPath := ExpandConstant('{%CONTENT_ROOT}')
+    systemKaliteDir := ExpandConstant(userPath + '\.kalite')
+    userKaliteContent := ExpandConstant(systemKaliteDir + '\content');
+    if Not DirExists(userKaliteContent) then
+    begin
+     if Not DirExists(contentPath) then
+        begin

```

Comment:
Hi @benjaoming if we remove this condition we also need to remove this `'KA Lite Setup cannot find any video contents in the default location (' + contentPath + ').` 

 In this sentence  
`'KA Lite Setup cannot find any video contents in the default location (' + contentPath + '). You can download contents after the installation or select a location with previously downloaded contents. If you don't have a folder with downloaded contents, press Cancel. To change the default location of downloaded contents, press OK.'`

We also can't say if there's a content folder but it's empty, unless we check the file size of it.

This will always ask the user set there content path during installation but it's optional.